src/components/DearVolunteer/DearVolunteer.module.css

```diff
@@ -0,0 +1,96 @@
+.dearVolunteer {
+  display: flex;
+  flex-direction: column;
+  padding: 64px 20px 64px 20px;
+  align-items: center;
+  gap: 32px;
+}
+.container {
+  display: flex;
+  flex-direction: column;
+  gap: 32px;
+  align-items: center;
+}
+.textWrapper {
+  display: flex;
+  flex-direction: column;
+  gap: 44px;
+}
+.textBox1 {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+.textBox2 {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  align-items: center;
+}
```

Comment:
textBox1 i 2 się nie różnią za wyjątkiem align-items, które nic nie zmienia. Śmiało można połączyć to w jedną klasę.